consumers/src/main/java/uk/gov/moj/sdt/consumers/config/ConsumersConfig.java

```diff
@@ -42,32 +37,17 @@ public class ConsumersConfig {
-                                                                                              GenericEnricher submitQueryRequestEnricher,
-                                                                                      @Qualifier("IndividualRequestEnricher")
-                                                                                              GenericEnricher individualRequestEnricher) {
-
-
-        XmlOutboundInterceptor xmlOutboundInterceptor = new XmlOutboundInterceptor();
-        List<AbstractSdtEnricher> enricherList = new ArrayList<>();
-        enricherList.add(submitQueryEnricher);
-        enricherList.add(bulkFeedbackEnricher);
-        enricherList.add(submitQueryRequestEnricher);
-        enricherList.add(individualRequestEnricher);
-        xmlOutboundInterceptor.setEnricherList(enricherList);
-
-
-        CacheSetupOutboundInterceptor cacheSetupOutboundInterceptor = new CacheSetupOutboundInterceptor();
-        PerformanceLoggerOutboundInterceptor performanceLoggerOutboundInterceptor = new PerformanceLoggerOutboundInterceptor();
-        CacheEndOutboundInterceptor cacheEndOutboundInterceptor = new CacheEndOutboundInterceptor();
-        PerformanceLoggerInboundInterceptor performanceLoggerInboundInterceptor = new PerformanceLoggerInboundInterceptor();
-        XmlInboundInterceptor xmlInboundInterceptor = new XmlInboundInterceptor();
-        SdtUnmarshallInterceptor sdtUnmarshallInterceptor = new SdtUnmarshallInterceptor();
-
+    public ITargetAppInternalEndpointPortType createTargetAppInternalEndpointPortType(@Qualifier("XmlOutboundInterceptor")
+                                                                                          XmlOutboundInterceptor xmlOutboundInterceptor,
+                                                                                      @Qualifier("CacheSetupOutboundInterceptor")
+                                                                                      CacheSetupOutboundInterceptor cacheSetupOutboundInterceptor,
+                                                                                      @Qualifier("CacheEndOutboundInterceptor")
+                                                                                          CacheEndOutboundInterceptor cacheEndOutboundInterceptor,
+                                                                                      @Qualifier("XmlInboundInterceptor")
+                                                                                          XmlInboundInterceptor xmlInboundInterceptor,
+                                                                                      @Qualifier("SdtUnmarshallInterceptor")
+                                                                                          SdtUnmarshallInterceptor sdtUnmarshallInterceptor
+                                                                                      ) {
```

Comment:
I think the Qualifier annotations require the corresponding classes to be annotated with Component, but they don't seem to be, e.g. the XMLOutboundInterceptor class isn't annotated with `@Component("XmlOutboundInterceptor")`.  The main branch has this change in, plus the annotated classes.  I think this branch needs to be updated with the changes from the main branch.